draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -320,18 +320,19 @@ verification key for the Issuer-signed JWT:
 
 - JWT Issuer Metadata: If the `iss` value contains an HTTPS URI, the recipient MUST
 obtain the public key using JWT Issuer Metadata as defined in (#jwt-issuer-metadata).
+- X.509 Certificates: The recipient MUST obtain the public key from the leaf X.509 certificate
+corresponding to the `x5c`, `x5c`, `x5t`, or `x5t#S256` JWT header parameters of the Issuer-signed JWT if one of them is present. In this case, the recipient MUST validate the X.509 certificate chain and the recipient MUST validate the `iss` value as follows:
```

Comment:
`x5t` and `x5t#S256` are just the hash of a certificate and presume the recipient has some way of finding the  certificate referenced. So it's a little bit awkward to talk about them like this as though they are basically the same as `x5u`/`x5c`.

Reply:
There are SSRF concerns with `x5u` too...

Reply:
... so maybe fewer options would be better/appropriate?

Reply:
- `x5u` could indeed lead to some weird vulnerability if verifiers are not careful enough and don't validate the entire X509 certificate chain which includes trusting the root certificate or the certificate itself in case it is self-signed.
- It is indeed the assumption that the verifier has to get the certificates out-of-band or side loaded somehow if `x5t` and `x5t#S256` as used.

I added the other x5* because I thought we had used them in OpenID4VP as well but after double-checking it seems that got removed and only `x5c` is now being used there. I think it is reasonable to only include `x5c`. I'll update the PR accordingly.